[yamaha] amplifier remain offline after openhab restart

## description
My Yamaha receiver works as expected, when it is switched on during openhab restart. When the receiver is switched off, the receiver remains offline in openhab after an openhab restart.
A bundle restart solves this problem then.

At startup this messages are written to the log. After switching ON the yamaha amplifier nothing happens, till I restart the bundle. I fear this could be bug. Hope someone can have a look at it. Thanks.

```
2019-04-12 21:25:39.666 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading model 'yamaha.things'

2019-04-12 21:25:55.089 [INFO ] [.dashboard.internal.DashboardService] - Started Dashboard at http://192.168.2.15:8080
2019-04-12 21:25:55.100 [INFO ] [.dashboard.internal.DashboardService] - Started Dashboard at https://192.168.2.15:8443
2019-04-12 21:25:57.655 [WARN ] [internal.handler.YamahaBridgeHandler] - Communication error. Please review your Yamaha thing configuration.
java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at java.net.Socket.connect(Socket.java:538) ~[?:?]
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558) ~[?:?]
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:242) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:339) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:357) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1050) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1334) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1309) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.postMessage(XMLConnection.java:81) ~[247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.sendReceive(XMLConnection.java:119) ~[247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getResponse(XMLProtocolService.java:81) ~[247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.DeviceInformationXML.update(DeviceInformationXML.java:91) ~[247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.updateDeviceInformation(YamahaBridgeHandler.java:258) ~[247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.connectionEstablished(YamahaBridgeHandler.java:345) [247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolFactory.createConnection(XMLProtocolFactory.java:30) [247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.initialize(YamahaBridgeHandler.java:325) [247:org.openhab.binding.yamahareceiver:2.5.0.201904040140]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [135:org.openhab.core:2.5.0.201904010543]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [135:org.openhab.core:2.5.0.201904010543]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
2019-04-12 21:25:59.386 [WARN ] [core.thing.internal.ThingManagerImpl] - Initializing handler for thing 'yamahareceiver:yamahaAV:receiverWZ' takes more than 5000ms.
2019-04-12 21:26:00.727 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler@7347952a': Could not communicate with the device
java.lang.RuntimeException: Could not communicate with the device
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:62) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolFactory.InputConverter(XMLProtocolFactory.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.initialize(YamahaBridgeHandler.java:326) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [135:org.openhab.core:2.5.0.201904010543]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [135:org.openhab.core:2.5.0.201904010543]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at java.net.Socket.connect(Socket.java:538) ~[?:?]
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558) ~[?:?]
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:242) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:339) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:357) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1050) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1334) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1309) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.postMessage(XMLConnection.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.sendReceive(XMLConnection.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getResponse(XMLProtocolService.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getZoneResponse(XMLProtocolService.java:69) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getInputs(XMLProtocolService.java:101) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.createInputsWithoutMapping(InputConverterXML.java:100) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:59) ~[?:?]
        ... 12 more
2019-04-12 21:26:00.776 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'yamahareceiver:yamahaAV:receiverWZ': Could not communicate with the device
java.lang.RuntimeException: Could not communicate with the device
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:62) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolFactory.InputConverter(XMLProtocolFactory.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.initialize(YamahaBridgeHandler.java:326) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [135:org.openhab.core:2.5.0.201904010543]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [135:org.openhab.core:2.5.0.201904010543]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at java.net.Socket.connect(Socket.java:538) ~[?:?]
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558) ~[?:?]
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:242) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:339) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:357) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1050) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1334) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1309) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.postMessage(XMLConnection.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.sendReceive(XMLConnection.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getResponse(XMLProtocolService.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getZoneResponse(XMLProtocolService.java:69) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getInputs(XMLProtocolService.java:101) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.createInputsWithoutMapping(InputConverterXML.java:100) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:59) ~[?:?]
        ... 12 more

```

My thing definition:
```
Bridge yamahareceiver:yamahaAV:receiverWZ "Yamaha Receiver RX-V473" [host="192.168.x.yyy"] {
	Thing zone Main_Zone "Main Zone" @ "Wohnzimmer" [zone="Main_Zone", volumeDbMin=-81, volumeDbMax=12]
}
```

## Expected Behavior
I would expect, that the binding discoveres automatically that the amplifier is switched on and initializes the thing.

## Current Behavior
As explained above:
the amplifier stay offline, when it was switched off during openhab restart.

## Possible Solution
As workaround: restart the yamaha binding

## Steps to Reproduce (for Bugs)
0. yamaha amplifier thing is online (e.g. in PaperUI)
1. switch off the Yamaha amplifier
2. restart openhab
3. switch on yamaha amplifier
3a. check: yamaha amplifier thing stays offline in openhab
4. bundle:restart <yamaha binding>
4a. check:  yamaha amplifier thing is online (again) in openhab

## Your Environment
openhab2                         2.5.0~S1566-1
debian stretch

This messages I get at openhab restart:
```
2019-04-24 10:25:41.731 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler@1d5650ff': Could not communicate with the device
java.lang.RuntimeException: Could not communicate with the device
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:62) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolFactory.InputConverter(XMLProtocolFactory.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.initialize(YamahaBridgeHandler.java:326) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [135:org.openhab.core:2.5.0.201904010543]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [135:org.openhab.core:2.5.0.201904010543]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at java.net.Socket.connect(Socket.java:538) ~[?:?]
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558) ~[?:?]
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:242) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:339) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:357) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1040) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1038) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:782) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1037) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1334) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.access$100(HttpURLConnection.java:91) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$8.run(HttpURLConnection.java:1301) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$8.run(HttpURLConnection.java:1299) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:782) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1298) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.postMessage(XMLConnection.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.sendReceive(XMLConnection.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getResponse(XMLProtocolService.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getZoneResponse(XMLProtocolService.java:69) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getInputs(XMLProtocolService.java:101) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.createInputsWithoutMapping(InputConverterXML.java:100) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:59) ~[?:?]
        ... 12 more

==> /var/log/openhab2/events.log <==
2019-04-24 10:25:41.763 [hingStatusInfoChangedEvent] - 'yamahareceiver:yamahaAV:receiverWZ' changed from OFFLINE (COMMUNICATION_ERROR): Keine Route zum Zielrechner (Host unreachable) to UNINITIALIZED (HANDLER_INITIALIZING_ERROR): Could not communicate with the device

==> /var/log/openhab2/openhab.log <==
2019-04-24 10:25:41.761 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'yamahareceiver:yamahaAV:receiverWZ': Could not communicate with the device
java.lang.RuntimeException: Could not communicate with the device
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:62) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolFactory.InputConverter(XMLProtocolFactory.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.handler.YamahaBridgeHandler.initialize(YamahaBridgeHandler.java:326) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [135:org.openhab.core:2.5.0.201904010543]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [135:org.openhab.core:2.5.0.201904010543]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at java.net.Socket.connect(Socket.java:538) ~[?:?]
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463) ~[?:?]
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558) ~[?:?]
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:242) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:339) ~[?:?]
        at sun.net.www.http.HttpClient.New(HttpClient.java:357) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1040) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1038) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:782) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1037) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1334) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.access$100(HttpURLConnection.java:91) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$8.run(HttpURLConnection.java:1301) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$8.run(HttpURLConnection.java:1299) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:782) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1298) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.postMessage(XMLConnection.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConnection.sendReceive(XMLConnection.java:119) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getResponse(XMLProtocolService.java:81) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getZoneResponse(XMLProtocolService.java:69) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLProtocolService.getInputs(XMLProtocolService.java:101) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.createInputsWithoutMapping(InputConverterXML.java:100) ~[?:?]
        at org.openhab.binding.yamahareceiver.internal.protocol.xml.InputConverterXML.<init>(InputConverterXML.java:59) ~[?:?]
        ... 12 more

```
Is there nobody who can have a look at this?

--- a/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaBridgeHandler.java
+++ b/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaBridgeHandler.java
@@ -236,6 +236,11 @@ public class YamahaBridgeHandler extends BaseBridgeHandler
             return;
         }
 
+        if (!ensureConnectionInitialized()) {
+            // The initialization did not yet happen and the device is still offline (or not reachable)
+            return;
+        }
+
         logger.trace("updateAllZoneInformation");
         try {
             // Set power = true before calling systemControl.update(),
@@ -260,8 +265,8 @@ public class YamahaBridgeHandler extends BaseBridgeHandler
                 }
             }
         } catch (IOException e) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
             systemControlState.invalidate();
+            onConnectivityError(e);
             return;
         } catch (ReceivedMessageParseException e) {
             updateProperty(PROPERTY_MENU_ERROR, e.getMessage());
@@ -272,20 +277,8 @@ public class YamahaBridgeHandler extends BaseBridgeHandler
         }
     }
 
-    private void updateDeviceInformation() throws IOException, ReceivedMessageParseException {
-        logger.trace("updateDeviceInformation");
-
-        DeviceInformation deviceInformation = protocolFactory.DeviceInformation(connection, deviceInformationState);
-        deviceInformation.update();
-
-        updateProperty(PROPERTY_VERSION, deviceInformationState.version);
-        updateProperty(PROPERTY_ASSIGNED_NAME, deviceInformationState.name);
-
-        zoneDiscoveryService.publishZones(deviceInformationState, thing.getUID());
-    }
-
     /**
-     * We handle the update ourself to avoid a costly dispose/initialize
+     * We handle the update ourselves to avoid a costly dispose/initialize
      */
     @Override
     public void handleConfigurationUpdate(Map<String, Object> configurationParameters) {
@@ -311,7 +304,7 @@ public class YamahaBridgeHandler extends BaseBridgeHandler
         Optional<String> host = bridgeConfig.getHostWithPort();
         if (host.isPresent()) {
             connection.setHost(host.get());
-            connectionEstablished(connection);
+            onConnectionCreated(connection);
         }
 
         inputConverter = protocolFactory.InputConverter(connection, bridgeConfig.getInputMapping());
@@ -355,40 +348,62 @@ public class YamahaBridgeHandler extends BaseBridgeHandler
             return;
         }
 
-        if (this.zoneDiscoveryService == null) {
+        if (zoneDiscoveryService == null) {
             logger.warn("Zone discovery service not ready!");
             return;
         }
 
         protocolFactory.createConnection(host.get(), this);
-        inputConverter = protocolFactory.InputConverter(connection, bridgeConfig.getInputMapping());
     }
 
     @Override
-    public void connectionFailed(String host, Throwable throwable) {
-        if (throwable != null) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, throwable.getMessage());
-        } else {
-            updateStatus(ThingStatus.OFFLINE);
-        }
-        this.connection = null;
-    }
-
-    @Override
-    public void connectionEstablished(AbstractConnection connection) {
-        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, "Waiting for data");
+    public void onConnectionCreated(AbstractConnection connection) {
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, "Waiting for connection with Yamaha device");
 
         this.connection = connection;
-        try {
-            updateDeviceInformation();
-            systemControl = protocolFactory.SystemControl(connection, this, deviceInformationState);
-        } catch (IOException | ReceivedMessageParseException e) {
-            logger.warn("Communication error. Please review your Yamaha thing configuration.", e);
-            deviceInformationState.invalidate();
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
-            return;
+        this.systemControl = null;
+
+        if (!ensureConnectionInitialized()) {
+            logger.warn("Communication error. Please review your Yamaha thing configuration.");
         }
+
         setupRefreshTimer(0);
+    }
+
+    /**
+     * Attempts to perform a one-time initialization after a connection is created.
+     * @return true if initialization was successful
+     */
+    private boolean ensureConnectionInitialized() {
+        if (systemControl != null) {
+            return true;
+        }
+
+        logger.trace("Initializing connection");
+
+        try {
+            DeviceInformation deviceInformation = protocolFactory.DeviceInformation(connection, deviceInformationState);
+            deviceInformation.update();
+
+            updateProperty(PROPERTY_VERSION, deviceInformationState.version);
+            updateProperty(PROPERTY_ASSIGNED_NAME, deviceInformationState.name);
+
+            zoneDiscoveryService.publishZones(deviceInformationState, thing.getUID());
+
+            systemControl = protocolFactory.SystemControl(connection, this, deviceInformationState);
+            inputConverter = protocolFactory.InputConverter(connection, bridgeConfig.getInputMapping());
+        } catch (IOException | ReceivedMessageParseException e) {
+            deviceInformationState.invalidate();
+            onConnectivityError(e);
+            return false;
+        }
+        return true;
+    }
+
+    private void onConnectivityError(Exception e) {
+        String description = e.getMessage();
+        logger.debug("Communication error. Either the Yamaha thing configuration is invalid or the device is offline. Details: {}", description);
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, description);
     }
 
     @Override

--- a/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/ConnectionStateListener.java
+++ b/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/ConnectionStateListener.java
@@ -18,7 +18,5 @@ package org.openhab.binding.yamahareceiver.internal.protocol;
  * @author David Graeff - Initial contribution
  */
 public interface ConnectionStateListener {
-    void connectionFailed(String host, Throwable throwable);
-
-    void connectionEstablished(AbstractConnection connection);
+    void onConnectionCreated(AbstractConnection connection);
 }

--- a/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLConnection.java
+++ b/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLConnection.java
@@ -59,7 +59,7 @@ public class XMLConnection extends AbstractConnection {
     private <T> T postMessage(String prefix, String message, String suffix,
             CheckedConsumer<HttpURLConnection, T> responseConsumer) throws IOException {
         if (message.startsWith("<?xml")) {
-            throw new IOException("No preformatted xml allowed!");
+            throw new IOException("No pre-formatted xml allowed!");
         }
         message = prefix + message + suffix;
 
@@ -74,6 +74,7 @@ public class XMLConnection extends AbstractConnection {
             connection.setRequestMethod("POST");
             connection.setRequestProperty("Content-Length", Integer.toString(message.length()));
 
+            connection.setConnectTimeout(5); // set a timeout in case the device is not reachable (went offline)
             connection.setUseCaches(false);
             connection.setDoInput(true);
             connection.setDoOutput(true);

--- a/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLProtocolFactory.java
+++ b/bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLProtocolFactory.java
@@ -47,7 +47,7 @@ import org.openhab.binding.yamahareceiver.internal.state.ZoneControlStateListene
 public class XMLProtocolFactory implements ProtocolFactory {
     @Override
     public void createConnection(String host, ConnectionStateListener connectionStateListener) {
-        connectionStateListener.connectionEstablished(new XMLConnection(host));
+        connectionStateListener.onConnectionCreated(new XMLConnection(host));
     }
 
     @Override
@@ -132,8 +132,6 @@ public class XMLProtocolFactory implements ProtocolFactory {
 
     @Override
     public InputConverter InputConverter(AbstractConnection connection, String setting) {
-
         return new InputConverterXML(connection, setting);
     }
-
 }

--- a/bundles/org.openhab.binding.yamahareceiver/src/test/java/org/openhab/binding/yamahareceiver/internal/YamahaReceiverHandlerTest.java
+++ b/bundles/org.openhab.binding.yamahareceiver/src/test/java/org/openhab/binding/yamahareceiver/internal/YamahaReceiverHandlerTest.java
@@ -95,7 +95,7 @@ public class YamahaReceiverHandlerTest extends AbstractXMLProtocolTest {
         subject.setCallback(callback);
 
         doAnswer(a -> {
-            ((ConnectionStateListener) a.getArgument(1)).connectionEstablished(ctx.getConnection());
+            ((ConnectionStateListener) a.getArgument(1)).onConnectionCreated(ctx.getConnection());
             return null;
         }).when(protocolFactory).createConnection(anyString(), same(subject));
     }

